app/views/questions/_questions.html.erb

```diff
@@ -1,6 +1,7 @@
+<% questions = questions || @questions %>
```

Comment:
Yeah makes sense. But I am having trouble using the partials with pagination, because I have four different items on the same page..i.e `notes, wikis, questions and profiles`. And they are all trying to paginate the same page. But will try to find a work-around on that and if you have any pointers please let me know.